### PR TITLE
[Stories] Temporary gutenberg update for media files type

### PR DESCRIPTION
**Pull request by bjtitus**
Fixes an incorrect type in the GutenbergBridgeDelegate methods for the "media files" block (a.k.a. Stories).

See https://github.com/wordpress-mobile/gutenberg-mobile/pull/3113
See https://github.com/WordPress/gutenberg/pull/28766

### Testing

* Open a post containing a Story block.
* Attempt to edit the story.
* An error screen should not appear after this change has been made.

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -73,7 +73,7 @@ PODS:
   - GTMSessionFetcher/Core (1.5.0)
   - GTMSessionFetcher/Full (1.5.0):
     - GTMSessionFetcher/Core (= 1.5.0)
-  - Gutenberg (1.46.0):
```
Comment (guarani) [inline Podfile.lock]: Just came here from https://github.com/WordPress/gutenberg/pull/28766 and noticed that this is going down a version (1.46.0 to 1.45.0), could your Gutenberg Mobile or Gutenberg branches need updating?

**Comment by jkmassel** [comment]
👋 We're freezing `16.7` today, so this PR is being bumped to `16.8`. If you need this to be part of the `16.7` release, please merge it into the `release/16.7` branch and DM me – I'll be happy to cut a new beta release!

File: Podfile [hunk 2]
```diff
@@ -154,7 +154,8 @@ target 'WordPress' do
     ## Gutenberg (React Native)
     ## =====================
     ##
-    gutenberg :tag => 'v1.46.0'
+    #gutenberg  :tag => 'v1.46.0'
+    gutenberg  :commit => 'dcacde'
```
Comment (guarani) [inline Podfile]: I undid this change temporarily on my machine, ran `rake xcode` and ran the project – without the packager running – expecting to see the red error screen, and I didn't get an error when I opened the story post.

**Review by guarani — COMMENTED** [review]
👋 @bjtitus, I'm not sure if you're planning on merging this PR, or just letting the Gutenberg Mobile changes get incorporated into WPiOS in the next release. I guess your options include:
a. If you want these changes incorporated immediately into WPiOS `develop` (e.g. because you need to do more work that depends on this), you'll need to run `npm run bundle` in the gutenberg-mobile directory, commit the results and push to your [PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3113). Then you'll have to update this PR to point to that latest commit
b. However, if this change isn't needed rightaway (i.e. no PRs this week or next depend on it), you can use this PR only for testing. If this is the case, you can close this PR once the Gutenberg and Gutenberg Mobile PRs are ready and merged. In this scenario, this PR is only used for creating an App Center build with your Gutenberg changes.

Let me know if you need a hand with any of this.

File: Podfile (hunk 2 above)
Comment (guarani) [reply Podfile]: I didn't get an error either when running the packager on the Gutenberg Mobile branch, which is a good thing, but I expected to be able to reproduce the error by undoing the above change but wasn't successful.

**Review by guarani — APPROVED** [review]
Tested and looking good!

**Pull request merged**